extensions/amp-video/0.1/test/test-amp-video.js

```diff
@@ -187,7 +187,8 @@ describes.realWin('amp-video', {
     });
   });
 
-  it('should not load a video with http src', () => {
+  // TODO(#21767): re-enable this test.
+  it.skip('should not load a video with http src', () => {
```

Comment:
This test is most likely doing an async `throw()` that isn't being caught by the `expect` block.